File: dotnet/Multiplayer/Rooms/RoomKickResponse.cs
using Michitai.Multiplayer.Errors;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Michitai.Multiplayer.Rooms
{
    /// <summary>
    /// Response returned when a player is successfully kicked from a game room.
    /// </summary>
    public class RoomKickResponse : ApiResponse<ERoomKickError>
    {
        /// <summary>
        /// Confirmation message for the kick operation.
        /// </summary>
        public string Message { get; set; } = string.Empty;

        /// <summary>
        /// The ID of the player who was kicked.
        /// </summary>
        public int KickedPlayerId { get; set; }
    }
}

File: java/src/main/java/com/michitai/multiplayer/rooms/RoomKickResponse.java
package com.michitai.multiplayer.rooms;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.michitai.multiplayer.ApiResponse;

/**
 * Response confirming a player was kicked from the room.
 */
public class RoomKickResponse extends ApiResponse {
    @JsonProperty("message")
    private String message;

    @JsonProperty("player_id")
    private int playerId;

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public int getPlayerId() {
        return playerId;
    }

    public void setPlayerId(int playerId) {
        this.playerId = playerId;
    }
}
